ietf/templates/meeting/requests.html

```diff
@@ -32,21 +32,22 @@ <h2 class="mt-5" id="{{ area.grouper.acronym }}">
                     <th data-sort="group">Group</th>
                     <th data-sort="num">Length</th>
                     <th data-sort="num">Size</th>
-                    <th data-sort="requester">Requester</th>
-                    <th data-sort="ad">AD</th>
+                    <th class="d-none d-sm-table-cell" data-sort="requester">Requester</th>
+                    <th class="d-none d-sm-table-cell" data-sort="ad">AD</th>
                     <th data-sort="constraints">Constraints</th>
                     <th data-sort="special">Special requests</th>
                 </tr>
             </thead>
             {% for session in area.list %}
                 {% ifchanged %}
+                    {% if not forloop.first %}</tbody>{% endif %}
                     <thead>
                         <tr>
                             <th class="table-warning" colspan="7">{{ session.current_status_name|capfirst }}</th>
                         </tr>
                     </thead>
+                    <tbody>
```

Comment:
Isn't this going to leave an unclosed `<tbody>` at the last iteration of the loop?